recipes/mariadb/LICENSE

```diff
@@ -0,0 +1,504 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
This file is already included in the PyPI zip file, so you don't need to (and should not) include it here. Please remove.

Reply:
Removed License.